### ISSUE TITLE
Question/Issue: No way to check if program works

### ISSUE THREAD

**Issue by lukisdbc** [broderickhyman/albiondata-client#125]
Hello, I would love to help you to collect data using your program, but It's not clear how to use it and to check if I use it correctly. There is no any Readme or instruction, only .exe in Release folder (readme.md in sources doesn't contain any instruction). I installed it, I have App in start that I can click to run. But what next? There is no app running anywhere i can find, so I dont even know if it started successfuly or not, I dont know if its working or I didnt run it or is it throwing some exceptions. I can't find also it in task manager.

IN SHORT : Please make some instruction clearer so its more accessable and so people can more easily run your client and help you to collect data.

**Comment by lukisdbc** [comment]
I tried to test if my client works by finding data for resource that was not available from API and then went to market in albion to get info about this resource. After hour some of resources are available on API and some not, I can only guess if my data was partially uploaded or its someone else or some else scenario happened.

**Comment by lukisdbc** [comment]
**EDIT** : I had to run program as administrator, now i see the commandline window that application is running. Please add this information somewhere so other people won't have to discover it like me.
